### PR TITLE
Add pages

### PR DESCRIPTION
**Pull request by DCalliet**
@dborstelmann

### CODE REVIEW

File: civiwiki/api/urls_templates.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import views_templates as v
 
 urlpatterns = [
+    url(r'^createpage', v.create_page, name='add page'),
```
Comment (dborstelmann) [inline civiwiki/api/urls_templates.py:5]: What does name even do, I never knew.  Are spaces a good idea?

File: civiwiki/civiwiki/urls.py [hunk 2]
```diff
@@ -25,7 +25,8 @@
     url(r'^api/', include(urls_write)),
     url(r'^api/', include(urls_read)),
     url(r'^api/', include(urls_auth)),
-    url(r'^',include(urls_templates)),
+    url(r'^/',include(urls_templates)),
+    url(r'^$',include(urls_templates)),
```
Comment (dborstelmann) [inline civiwiki/civiwiki/urls.py:29]: Porque?

Comment (DCalliet) [inline civiwiki/civiwiki/urls.py:29]: So at first the way we had it was like a base case where that matched everything that didn't come before it. I want it to specifically match the empty string or a path that is in url templates. This way urls like /media and /static aren't ignored

**Comment by dborstelmann** [comment]
@DCalliet I didn't look too hard at add_page, I hope you tested that

File: civiwiki/api/urls_templates.py (hunk 1 above)
Comment (DCalliet) [inline civiwiki/api/urls_templates.py:5]: It's for reverse lookup if we ever have a namespace conflict from different apps, doubt we will get to that case so now it's just a descriptor

File: civiwiki/civiwiki/urls.py (hunk 2 above)
Comment (dborstelmann) [inline civiwiki/civiwiki/urls.py:29]: but why the $

Comment (DCalliet) [inline civiwiki/civiwiki/urls.py:29]: Regex my friend. This is matching the empty string but not anything else, without that end tag it matches everything.

Comment (dborstelmann) [inline civiwiki/civiwiki/urls.py:29]: gotcha

**Pull request merged**